plugin/net/driver.go

```diff
@@ -189,11 +200,30 @@ func (driver *driver) findNetworkInfo(id string) (network, error) {
 	if found {
 		return network, nil
 	}
+
+	// plugin-v2 does not have access to docker.sock, so we cannot call Docker
+	// API for the network info.
+	if driver.isPluginV2 {
+		// safe to set, as isOurs used only by the watcher which is disabled for plugin-v2
+		network.isOurs = true
+		network.hasMulticastRoute = driver.forceMulticast
+
+		driver.Lock()
```

Comment:
I think I might have omitted adding the data to the map, since we would just end up back here next time in that case.
Locking/adding in two places makes me nervous.